srcpkgs/dotnet-runtime/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'dotnet-runtime'
+pkgname=dotnet-runtime
```

Comment:
It seems the package just repackages precompiled binaries, in this case, the pkgname should be suffixed with `-bin`